File: go/raindrops/raindrops.go
package raindrops

import "strconv"

// Convert converts an integer into a string
func Convert(num int) string {
	returnStr := ""
	if num%3 == 0 {
		returnStr += "Pling"
	}
	if num%5 == 0 {
		returnStr += "Plang"
	}

	if num%7 == 0 {
		returnStr += "Plong"
	}

	if returnStr == "" {
		returnStr = strconv.Itoa(num)
	}
	return returnStr
}

File: python/raindrops/raindrops.py
def raindrops(number):
    s = ""
    if  number % 3 == 0:
        s += "Pling"
    if number % 5 == 0:
        s += "Plang"
    if number % 7 == 0 :
        s += "Plong"
    if  len(s) <= 0:
        s = str(number)
    return s

